roles/kubernetes-apps/ansible/tasks/netchecker.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Kubernetes Apps | Check AppArmor status
-  command: which apparmor_parser
+  command: grep Y /sys/module/apparmor/parameters/enabled
```

Comment:
How about using `aa-status` command instead?
The configuration path could be different between Linux distributions.
`aa-status` command returns the exit code `0` when apparmor is enabled like
```
       Upon exiting, aa-status will set its return value to the following values:

       0   if apparmor is enabled and policy is loaded.

       1   if apparmor is not enabled/loaded.

       2   if apparmor is enabled but no policy is loaded.

       3   if the apparmor control files aren't available under /sys/kernel/security/.

       4   if the user running the script doesn't have enough privileges to read the apparmor control
           files.
```